cosmos/job/drm/drm_k8s_jobs.py

```diff
@@ -138,3 +137,7 @@ def kill(self, task):
 
         if err:
             raise RuntimeError(err)
+    
```

Comment:
quick comment about why killing is necessary is the cleanup

explicit None checking is a bit better: `if task.drm_jobID is not None`